pkg/certificate/providers/certmanager/certificate_manager.go

```diff
@@ -242,45 +127,29 @@ func (cm *CertManager) issue(cn certificate.CommonName, validityPeriod time.Dura
 		}
 	}()
 
-	cm.cacheLock.Lock()
-	defer cm.cacheLock.Unlock()
-	cm.cache[cert.GetCommonName()] = cert
-
 	return cert, nil
 }
 
-// NewCertManager will construct a new certificate.Certificate implemented
+// New will construct a new certificate.Certificate implemented
```

Comment:
To match the function signature:
```suggestion
// New will construct a new CertManager implemented
```

Reply:
Changed the comment a bit. In general a struct implementing an interface doesn't need to know what interface it is implementing. This fit's the go paradigm: "define interfaces where they are consumed not where they are implemented"